pkg/capabilities/consensus/ocr3/capability.go

```diff
@@ -288,11 +279,6 @@ func (o *capability) expiryTimer(ctx context.Context, r *request) {
 			},
 		}
 		o.lggr.Debugw("expiryTimer - expired request", "workflowExecutionID", r.WorkflowExecutionID)
-		o.transmitCh <- resp
+		o.handleTransmitMsg(ctx, resp)
```

Comment:
I am unable to do a proper suggestion comment, but it looks like the context passed here is not actually used, so we could drop it and just receive from `o.stopCh` directly.
```go
func (o *capability) expiryTimer(r *request) {
	defer o.wg.Done()

	d := r.ExpiresAt.Sub(o.clock.Now())
	tr := o.clock.NewTimer(d)
	defer tr.Stop()
	select {
	case <-o.StopCh:
		return
	case <-tr.Chan():
		resp := &outputs{
			WorkflowExecutionID: r.WorkflowExecutionID,
			CapabilityResponse: capabilities.CapabilityResponse{
				Err:   fmt.Errorf("timeout exceeded: could not process request before expiry %s", r.WorkflowExecutionID),
				Value: nil,
			},
		}
		o.lggr.Debugw("expiryTimer - expired request", "workflowExecutionID", r.WorkflowExecutionID)
		o.handleTransmitMsg(resp)
``